ops/model.py

```diff
@@ -279,7 +290,13 @@ def __init__(self, name, relation_id, backend):
     @property
     def network(self):
         if self._network is None:
-            self._network = Network(self._backend.network_get(self.name, self._relation_id))
+            try:
+                self._network = Network(self._backend.network_get(self.name, self._relation_id))
```

Comment:
Given how much we already know of the upcoming feature, we should discuss whether to go forward with Network, or rename this to something related to "spaces". Should probably catch up with the juju team on this to avoid terminology discrepancies.

Reply:
@jameinel @mitechie ^

Reply:
This isn't a blocker for this PR though (as `Network` is already exposed)